.idea/Leaf-API-Postman-Collection.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```

Comment:
Is this file necessary? This is new for me

Reply:
woops, that wasn't intended. https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/60/commits/36602d4203e375b2c569a6575e0b4b0d17e3e559